### ISSUE TITLE
Is this a "Token.Literal.String.Doc"?

### ISSUE THREAD

**Issue by echoechoin** [pygments/pygments#2051]
```python
str = (
    """
    This is a test string.
    """
)
```

after `print(token)`:

```sh
(Token.Literal.String.Doc, '"""\n    This is a test string.\n    """')
```

**Comment by echoechoin** [comment]
using `PythonLexer`.

**Comment by echoechoin** [comment]
```
str = ("""
    This is a test string.
    """
)
```
But this is `Token.Literal.String.Double`. So I will reopen this issue.
```
(Token.Literal.String.Double, '"""')
(Token.Literal.String.Double, 'this is a test string')
(Token.Literal.String.Double, '\n')
(Token.Literal.String.Double, '"""')
```

**Comment by birkenfeld** [comment]
I guess the issue is that the string in the origin message should not be considered a docstring.

However, fixing this would need to switch from a simple heuristic to lexing class and function signatures completely separately, which we currently do not do. I think this is an acceptable error.

**Comment by echoechoin** [comment]
> I guess the issue is that the string in the origin message should not be considered a docstring.
> 
> However, fixing this would need to switch from a simple heuristic to lexing class and function signatures completely separately, which we currently do not do. I think this is an acceptable error.

Yes, if someone using `pygments` as code highlight library, this is an acceptable error indeed.
But I using this lib to do something related token type, so ... 😶‍🌫️
If you members don't want to fix it, just close this issue.

**Comment by Jean-Abou-Samra** [comment]
The goal of Pygments is to highlight code. What is your use case? I suspect it could use better tools, starting with Python's ast module.

**Comment by echoechoin** [comment]
I want to get both Python's token and Bash's token. So I found this lib and this little issue. Never mind, I have added some control code in my project and resolved my problem.
